### PR TITLE
2819/get assistance page

### PR DESCRIPTION
**Pull request by adriencyberspace**
# Pull Request Template

## Issue Overview

This PR addresses #2819

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## QA Notes:
Entry point: http://localhost:3000/ - any page

1. Notice Get Assistance page in navigation bar instead of Resources
2. Visit page (/get-assistance) and compare layout to Figma mockup
3. Click through links and ensure they link correctly

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by emilyjablonski** [comment]
Adrien and I did an in-person PR review. There are few questions for Jesse and a few minor code cleanups and then LGTM!

File: sites/public/src/ResourceLinkCard.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  linkLabel: string
+}
+
+const ResourceLinkCard = (props: ResourceLinkCardProps) => {
+  const { iconSymbol, title, subtitle, linkUrl, linkLabel } = props
+
+  return (
+    <div className="border-b">
+      <div className="my-6 md:my-12">
+        <Icon
```
Comment (emilyjablonski) [inline sites/public/src/ResourceLinkCard.tsx:19]: `<Icon symbol="house" size="2xl" className="border-primary border-b-4 inline-block pb-3 px-3 mb-2" />`
Here is what the Icon looks like from the comparable one in Alameda

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
All the changes we chatted about look great to me! I am seeing one regression on mobile where we lost horizontal padding, other than that lgtm

Screen Shot 2022-07-12 at 3 22 16 PM

**Comment by adriencyberspace** [comment]
@emilyjablonski Good call. I removed the section padding bc it was doubling the component padding on some screen widths, and since the Contact aside is used on multiple pages and has built-in padding (and so do other reusable components i.e. MarkdownSection) - I figured this would be a better solution for now. LMK if that works for you!

**Pull request merged**